src/sysext/sysext.c

```diff
@@ -474,11 +518,38 @@ static int verb_status(int argc, char **argv, void *userdata) {
         return ret;
 }
 
+static int append_overlayfs_path_option(
+                char** options,
+                const char* separator,
+                const char* option,
+                const char* path) {
+
+        _cleanup_free_ char *escaped = NULL;
+
+        assert (options);
+        assert (separator);
+        assert (path);
```

Comment:
coding style: the space between assert and `(` shouldn't be there

Reply:
Fixed.